### ISSUE TITLE
Compatibility with v13?

### ISSUE THREAD

**Issue by romanliedtke** [plan2net/webp#107]
Are there any plans to make the extension compatible with TYPO3 v13?

**Comment by rubensilva09** [comment]
Hi,

I've also come here to see if anything was planned for v13. Since no branch seem to have been created, I've forked the repo and have some changes ready for a pull request, but since there's no v13-support branch I'd rather wait for it to be created.

**Comment by wazum** [comment]
Yes, folks, switch to this extension and we will archive webp … less work for me, that's fine.

**Comment by Hasan-Shaker** [comment]
#110

**Comment by DenisMir** [comment]
An update to 13LTS would be nice. The above linked extension is buggy as hell.

**Comment by aggrabbah** [comment]
I've done it guys. Sorry for the wait, I just submitted my version for PR.
My colleague submitted an older version with some weird code that I forgot to remove.
I'll try to stick around longer so we can have this version up and running
